homeassistant/components/daikin/sensor.py

```diff
@@ -44,6 +47,9 @@ async def async_setup_entry(hass, entry, async_add_entities):
         sensors.append(ATTR_TOTAL_POWER)
         sensors.append(ATTR_COOL_ENERGY)
         sensors.append(ATTR_HEAT_ENERGY)
+    if daikin_api.device.values.get(ATTR_HUMIDITY).replace("-", ""):
```

Comment:
Same as above, this logic should be in pydaikin in a `support_humidity` property